theano/printing.py

```diff
@@ -268,13 +269,13 @@ def clone(self, props=None, **more_props):
             props = {}
         return PrinterState(self, **dict(props, **more_props))
 
-
```

Comment:
At the top level of a file, we need 2 empty line (pep8). So the old code was good.